### DISCUSSION TITLE
generateDepictionMatching2DStructure failed

### DISCUSSION THREAD

**Discussion by wanghmalan** [rdkit/rdkit#5769] [Q&A] [answered]
i want to draw a smile with generateDepictionMatching2DStructure

```
smi = "O=C(C1=CN(C2=NC=CS2)C3=C(C=CC(Cl)=N3)C1=O)OCC"
template_smi = "C12=CC=CN=C1NC=CC2'

template = Chem.MolFromSmiles(template_smi)
AllChem.Compute2DCoords(template)
m = Chem.MolFromSmiles(smi)
AllChem.GenerateDepictionMatching2DStructure(m, template)
```

but it raise Depict error: Substructure match with reference not found.
why template_smi and smi not match?

**Comment by greglandrum** [comment, marked as answer]
@wanghmalan : the central ring system in your molecule is considered to be aromatic by the RDKit, but the template molecule is not, so there's no substructure match. If you change your template to `C12=CC=CN=C1NC=CC2=O` everything should work.

**Reply by wanghmalan** [reply]
thanks for answer~
